src/component/page/deltakerliste-veileder/FilterMenyDeltakerliste.tsx

```diff
@@ -0,0 +1,58 @@
+import { Checkbox, CheckboxGroup, Panel } from '@navikt/ds-react'
+import React from 'react'
+
+import { useTiltaksoversiktSokStore } from '../../../store/tiltaksoversikt-sok-store'
+import { klikkFilterMeny, loggKlikk } from '../../../utils/amplitude-utils'
+import styles from './FilterMenyDeltakerliste.module.scss'
+
+interface Props {
+	deltakerlisteMap: Map<string, number>
+	className?: string
+}
+
+export const FilterMenyDeltakerliste = (props: Props): React.ReactElement => {
+	const {
+		deltakerlisteFilter,
+		leggTilDeltakerliste,
+		fjernFraDeltakerliste,
+	} = useTiltaksoversiktSokStore()
+
+	const unikeNavn =[ ...props.deltakerlisteMap.keys() ]
+
+	const DeltakerlisteCheckbox = ({ navn } : { navn: string }) => {
+		const antallDeltakere = props.deltakerlisteMap.get(navn) ?? 0
+
+		return  (
+			<Checkbox
+				className={styles.checkbox}
+				name="filter-deltakerliste"
+				checked={deltakerlisteFilter.includes(navn)}
+				onChange={(e) => {
+					if (e.target.checked) {
+						leggTilDeltakerliste(navn)
+						loggKlikk(klikkFilterMeny, navn, 'checked')
```

Comment:
Denne loggingen bør få et annet navn så man kan se forskjell på om man trykker på denne eller den andre filtermenyen.. Også er jeg litt usikker på om man ønsker å ta med navnet på deltakerlista her egentlig, er kanskje ikke så mye man kan gjøre med det i amplitude og kan være risikabelt mtp personvern for noen tiltakstyper

Reply:
Ønsker vi egentlig å logge dette til amplitude for alle filtertyper? Bruker vi det, og gir det verdi? Hvis ikke kan det kanskje bare fjernes?

Reply:
Kan evt fjerne også ja, kommer litt ann på hva man vil sjekke på amplitude men hadde kanskje foretrukket å beholde loggingen fordi det er ikke mulig å oppdrive historikken i etterkant

Reply:
Men da vil du ha en klikk-metode for amplitude pr filter da egentlig? "loggStatusFilterKlikk", "loggDeltakerlisteFilterKlikk", "loggVeiledertypeFilterKlikk"? Bare så jeg forstår og slipper å gjøre det feil :)

Reply:
Ja. Men kan være lurt å ikke rename den gamle(selv om navnet ikke helt passer nå) sånn at man får hele historikken i sammenheng og ikke må filtrere på to forskjellige navn

Reply:
Det kan jeg ordne.

Reply:
Ser ikke ut som at jeg trenger å duplisere hele den loggKlikk-metoden, det holder vel å endre inputen :)